draft-ietf-rtcweb-jsep.xml

```diff
@@ -3912,11 +3912,12 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
           <t>If the m= section proto value indicates use of RTP:
           <list style="symbols">
 
-            <t>If the m= section references any media formats, RTP
-            header extensions, or RTCP feedback mechanisms that were
-            not present in the corresponding m= section in the offer,
-            this indicates a negotiation problem and MUST result in an
-            error.</t>
+            <t>If the m= section references RTCP feedback mechanisms
+            that were not present in the corresponding m= section in
+            the offer, this indicates a negotiation problem and MUST
+            result in an error.  New RTP header extensions values are
+            allowed as described in <xref target="RFC5285" />, Section
+            6.</t>
```

Comment:
It seemed a point of confusion ... then again so was media types so glad to just add media types here too